modules/polling/context/BallotContext.tsx

```diff
@@ -409,18 +409,17 @@ export const BallotProvider = ({ children }: PropTypes): React.ReactElement => {
 
         const gaslessProvider = getGaslessProvider(network);
 
+        const voteTxCreator = () => getGaslessTransaction(gaslessProvider, gaslessTx.hash);
+        trackPollVote(voteTxCreator, getGaslessNetwork(network));
+
```

Comment:
I had moved this to wait for `mined` status previously because the `getGaslessTransction`function was returning an error when the tx was just sitting the queue because the provider can't find it. If you click the "View on Arbiscan" right away and the tx is in the queue, you'll see that the tx isn't found. But I guess 99% of the time it will be relayed shortly after so probably not an issue.

Reply:
Ah interesting.  Cool that makes sense to me